Apps/WebClient/src/NewClientApp/src/components/site/IdleComponent.vue

```diff
@@ -0,0 +1,60 @@
+﻿<script setup lang="ts">
+import { computed, ref, watch } from "vue";
+
+import IdleDialogComponent from "@/components/site/IdleDialogComponent.vue";
+import { Path } from "@/constants/path";
+import router from "@/router";
+import { useAuthStore } from "@/stores/auth";
+import { useConfigStore } from "@/stores/config";
+import { IdleDetector } from "@/utility/idleDetector";
+
+const authStore = useAuthStore();
+const configStore = useConfigStore();
+
+const idleDialog = ref<InstanceType<typeof IdleDialogComponent>>();
+
+const timeBeforeIdle = computed(() => configStore.webConfig.timeouts.idle);
+const maxIdleDialogCountdown = computed(() => 60000);
```

Comment:
```suggestion
const maxIdleDialogCountdown = 60000;
```

This will need changes where the `maxIdleDialogCountdown.value` is called. and the placement should be moved to the appropriate non reactive variables.